run_dir/design/project_samples.html

```diff
@@ -113,6 +113,7 @@
     <li><a href="#tab_running_notes_content" role="tab" data-toggle="tab">Running Notes</a></li>
     <li><a href="#tab_com_content" role="tab" data-toggle="tab" id="tab_communication">User communication</a></li>
     <li><a href="#tab_links_content" role="tab" data-toggle="tab">Links</a></li>
+    <li><a href="#tab_charon_data" role="tab" data-toggle="tab">Charon</a></li>
```

Comment:
Can we add `styel="display:none;"` to this, and then show it if there is any charon data to display?

```html
<li style="display:none;"><a href="#tab_charon_data" role="tab" data-toggle="tab">Charon</a></li>
```